src/DetailsView/components/failure-instance-panel-control.tsx

```diff
@@ -88,6 +97,62 @@ export class FailureInstancePanelControl extends React.Component<FailureInstance
             closeButtonAriaLabel: null,
         };
 
+        if (this.props.featureFlagStoreData && this.props.featureFlagStoreData[FeatureFlags.manualInstanceDetails]) {
```

Comment:
This is the right idea but we generally use `FlaggedComponent` in these scenarios. Look into examples of that in the code base and you should be able to use that instead.